docs/lib/graphqlapi/fragments/flutter/advanced-workflows/30_nested.md

```diff
@@ -0,0 +1,33 @@
+```dart
+try {
+    String graphQLDocument = '''query getPost(\$id: ID!) {
+        "getTodo(id: \$id) {
+            "id
+            "title
+            "rating
+            "status
+            "comments {
+                "items {
+                    "id
+                    "postID
+                    "content
+                }
+            }
+        }
+    }''';
+
+    var operation = Amplify.API.query<String>(
+        request: GraphQLRequest(
+            document: graphQLDocument,
+            variables: {"id": "[TODO_ID]"}
```

Comment:
Is the Id here an array?

Reply:
Unused file. Also this is a string (inside quotes) representation for a Todo id. Seems to be taken from the existing docs.